sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_first_seen_v1/metadata.yaml

```diff
@@ -7,7 +7,8 @@ labels:
   application: fxa
   incremental: true
   schedule: daily
-# # Descheduled for now. Last run was 2021-01-23.
+  referenced_tables: []  # exempts this query from dry runs
```

Comment:
`referenced_tables` only exists as part of `scheduling`. To exempt the query from dry runs, it would need to be added to `SKIP` in the `dryrun` script